sessionctx/variable/session.go

```diff
@@ -920,6 +928,10 @@ func (s *SessionVars) SetSystemVar(name string, val string) error {
 		s.DisableTxnAutoRetry = TiDBOptOn(val)
 	case TiDBEnableStreaming:
 		s.EnableStreaming = TiDBOptOn(val)
+	case TiDBEnableLargeTxn:
+		s.EnableLargeTxn = TiDBOptOn(val)
+	case TiDBLargeTxnSize:
+		s.LargeTxnSize = tidbOptInt64(val, DefTiDBLargeTxnSize)
```

Comment:
maybe we need to add `TiDBLargeTxnSize`  set value check in `ValidateSetSystemVar`